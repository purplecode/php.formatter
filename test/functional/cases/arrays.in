<?php

// messy array
$array = array(1 

	=> 

2

);

// should stay the same
$array = array('a' => array('b' => 'c'));


// should keep new lines
$array = array(
	'B' => array('b' => 'c', 'd' => 'e'),
	'A' => array('b' => 'c', 'd' => 'e')
));