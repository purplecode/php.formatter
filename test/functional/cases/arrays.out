<?php

$array = array(1 => 2);
